torch/csrc/Module.cpp

```diff
@@ -1546,6 +1575,7 @@ static PyMethodDef TorchMethods[] = { // NOLINT
      (PyCFunction)(void (*)())THPModule_has_torch_function_variadic,
      METH_FASTCALL,
      nullptr},
+    {"_c10_justknobs_check", THPModule_c10_justknobs_check, METH_VARARGS, nullptr},
```

Comment:
Can we update https://github.com/pytorch/pytorch/blob/605dfd8fb4672beef93b385ff91df5f349f2d394/torch/_utils_internal.py#L154 to use this?

Reply:
Ill update it, but I'd like to stack a PR that does because it will be that PR which breaks stuff while this one is pretty safe to land with the only use being the test.